### PR TITLE
UISEES-47: Validation error displayed when the user modifies the search value

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
[UISEES-47](https://issues.folio.org/browse/UISEES-47): Validation error displayed when the user modifies the search value

## Purpose
add edit mode to advanced search

### CODE REVIEW

File: src/components/ElasticQueryField/ElasticQueryField.js [hunk 1]
```diff
@@ -2,15 +2,36 @@ import React, { useState, useRef, useEffect } from 'react';
...
-import css from './ElasticQueryField.css';
 
-const UNSELECTED_OPTION_INDEX = -1;
-const SPACE = ' ';
-const OPEN_BRACKET = '(';
-const CLOSE_BRACKET = ')';
-const SLASH = '/';
+import getElasticQuery from './getElasticQuery';
+import {
+  addQuotes, changeTextAreaHeight,
```
Comment (Godlevskyi) [inline src/components/ElasticQueryField/ElasticQueryField.js]: Please format each import on separate line

File: src/components/ElasticQueryField/ElasticQueryField.js [hunk 2]
```diff
@@ -56,20 +75,37 @@ const ElasticQueryField = props => {
   const [options, setOptions] = useState([]);
   const [focusedOptionIndex, setFocusedOptionIndex] = useState(UNSELECTED_OPTION_INDEX);
   const [typedValue, setTypedValue] = useState('');
+  const [typedValueForEditMode, setTypedValue2] = useState('');
```
Comment (Godlevskyi) [inline src/components/ElasticQueryField/ElasticQueryField.js]: Please change the naming for the setter to be consistent 
const [typedValueForEditMode, setTypedValueForEditMode] = useState('');

File: src/components/ElasticQueryField/ElasticQueryField.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [reply src/components/ElasticQueryField/ElasticQueryField.js]: Thanks, fixed

File: src/components/ElasticQueryField/ElasticQueryField.js (hunk 2 above)
Comment (Dmytro-Melnyshyn) [reply src/components/ElasticQueryField/ElasticQueryField.js]: Thanks, fixed

**Review by Godlevskyi — APPROVED** [review]

**Pull request merged**